src/modules/Dropdown/Dropdown.js

```diff
@@ -184,21 +188,19 @@ export default class Dropdown extends Component {
 
   componentWillReceiveProps(nextProps) {
     super.componentWillReceiveProps(nextProps)
-    // console.debug('Dropdown.componentWillReceiveProps()')
-    // console.log('changed props:', _.transform(nextProps, (res, val, key) => {
-    //   if (val !== this.props[key]) res[key] = val
-    // }, {}))
+    debug.groupCollapsed('componentDidUpdate()')
+    debug('changed props:', objectDiff(nextProps, this.props))
+    debug.groupEnd()
```

Comment:
console methods `group`, `groupCollapsed`, and `groupEnd` are added for cleaner console activity.